### PR TITLE
st1 [2067][IMP] base_company_reporting, report_common_rpl: move bank_info

### PR DESCRIPTION
**Pull request by yostashiro**
[2067](https://www.quartile.co/web?debug=1#id=2067&action=771&model=project.task&view_type=form&menu_id=505)

Moving bank_info from base_company_reporting to report_common_rpl.

report_common_rpl was also not in a good shape, so some additional tweaks are done, including the removal of report_py3o from the dependency.

### CODE REVIEW

**Comment by yostashiro** [comment]
Pre-commit somehow fails, but it's passed locally.
image

```
(env) ytashiro@dev:~/ws/rpl/rpl-custom$ pre-commit run -a
black....................................................................Passed
prettier.................................................................Passed
prettier xml plugin......................................................Passed
eslint...................................................................Passed
- hook id: eslint
- duration: 0.9s
Trim Trailing Whitespace.................................................Passed
Fix End of Files.........................................................Passed
Debug Statements (Python)................................................Passed
flake8 except __init__.py................................................Passed
flake8 only __init__.py..................................................Passed
Fix python encoding pragma...............................................Passed
Check for case conflicts.................................................Passed
Check docstring is first.................................................Passed
Check that executables have shebangs.................(no files to check)Skipped
Check for merge conflicts................................................Passed
Check for broken symlinks............................(no files to check)Skipped
Check Xml................................................................Passed
Mixed line ending........................................................Passed
pylint with optional checks..............................................Passed
- hook id: pylint
- duration: 2.71s
pylint with mandatory checks.............................................Passed
pyupgrade................................................................Passed
isort except __init__.py.................................................Passed
```

File: report_common_rpl/__manifest__.py [hunk 1]
```diff
@@ -1,12 +1,12 @@
-# Copyright 2019 Quartile Limited
+# Copyright 2019-2022 Quartile Limited
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 {
-    "name": "Report Common Py3o",
-    "version": "12.0.1.0.0",
+    "name": "Report Common",
+    "version": "12.0.1.2.0",
```
Comment (tack-yabeQuartile) [inline report_common_rpl/__manifest__.py]: Do you have special meanings to skip the version "12.0.1.1.0"?

**Review by tack-yabeQuartile — APPROVED** [review]
LGTM, but I have a single comment (question).
please check.

File: report_common_rpl/__manifest__.py (hunk 1 above)
Comment (yostashiro) [reply report_common_rpl/__manifest__.py]: It was just a typo. Fixed now.

**Pull request merged**